File: Api/Timesheet/Entities/DailySchedule.cs
namespace Timesheet.Entities
{
    public class DailySchedule :EntityBase
    {

        public int DayEnum { get; set; }
        public DateTime InitialTime { get; set; }
        public DateTime EndTime { get; set; }
        public int ScheduleTypeId { get; set; }
        public ScheduleType? ScheduleType { get; set; }
    }
}

File: Web/src/app/models/records/daily-schedule.ts
import { ScheduleType } from "./shedule-type";
export class DailySchedule {
    Id: number;
    ScheduleTypeId: number;
    DayEnum: number;  
    InitialTime: string; 
    EndTime: string;
    ScheduleType?: ScheduleType;
  
    constructor(params?: Partial<DailySchedule>) {
      this.Id = params?.Id ;
      this.ScheduleTypeId = params?.ScheduleTypeId;
      this.DayEnum = params?.DayEnum;
      this.InitialTime = params?.InitialTime;
      this.EndTime = params?.EndTime ;
    }
  }
  
 
    
    